src/isomorphic/classic/element/ReactElement.js

```diff
@@ -167,8 +167,8 @@ ReactElement.createElement = function(type, config, children) {
         /* eslint-disable no-proto */
         config.__proto__ == null || config.__proto__ === Object.prototype,
         /* eslint-enable no-proto */
-        'React.createElement(...): Expected props argument to be a plain object. ' +
-        'Properties defined in its prototype chain will be ignored.'
+        'React.createElement(...): Expected props argument of ' + displayName(type) +
```

Comment:
Better to say 'argument of type X' as it's more in line with the language I've seen in other places